core/src/main/java/io/questdb/tasks/TelemetryWalTask.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.tasks;
+
+import io.questdb.Telemetry;
+import io.questdb.cairo.CairoException;
+import io.questdb.cairo.TableWriter;
+import io.questdb.log.Log;
+import io.questdb.log.LogFactory;
+import io.questdb.std.ObjectFactory;
+import org.jetbrains.annotations.NotNull;
+
+public class TelemetryWalTask extends AbstractTelemetryTask {
+    public static final String TABLE_NAME = "telemetry_wal";
```

Comment:
This is a system table. It should either have name starting from `config.getSystemTableNamePrefix` or be handled in table catalog functions.

Reply:
Fixed in https://github.com/questdb/questdb/pull/2930/commits/69d565260514a08087e2fd262f7576d00b5f45a5